src/ssh_audit/ssh_audit.py

```diff
@@ -947,19 +1002,21 @@ def process_commandline(out: OutputBuffer, args: List[str], usage_cb: Callable[.
         list_policies(out, aconf.verbose)
         sys.exit(exitcodes.GOOD)
 
-    if aconf.client_audit is False and aconf.target_file is None:
+    if aconf.client_audit is None and aconf.target_file is None:
         if oport is not None:
-            host = args[0]
+            host = host
```

Comment:
`host = host` is not needed.

Reply:
Of all of your comments these are the three I was expecting the most.

Each of the host = host ; port = port  was  me leaving the original code as untouched as possible; where ever it did not need to be touched.

Agree that this section can probably be adjusted and cleaned up.

The original host = args[0] was to take the trailing IP / hostname from the command line.
As I added the --ip arg to ensure I could trap the target IP I left the host =  and just changed it to host = host ; where the host variable is populated on line ~887

These could probably be changed to directly use the arguments similar to 

host = arguments.host
port = arguments.port

again this would be in an attempt to ensure minimal changes and unnecessary breakage on my part